examples/other/hirez_screenshot.py

```diff
@@ -0,0 +1,262 @@
+"""
+High resolution screenshot
+==========================
+
+"""
+
+# sphinx_gallery_pygfx_docs = 'code'
+# sphinx_gallery_pygfx_test = 'off'
+
+import os
+import tempfile
+import webbrowser
+
+import imageio as iio
+import numpy as np
+from rendercanvas.offscreen import RenderCanvas
+import pygfx as gfx
+
+
+# %% Prepare
+
+# The scale factor determines how many tiles are created to build the
+# final screenshot (upscale_factor**2). Using a higher value allows
+# creating screenshots above the wgpu texture size limit.
+upscale_factor = 4
+
+# Determine the canvas size, and thereby the resolution of each tile.
+# The tile_size and full_size that we calculate below are both in logical pixels.
+canvas_size = 1200, 1000
+
+if True:
```

Comment:
Could you put this bool in a variable so there's a name for the flag? It documents the logic a bit more clearly.

Reply:
done